controllers/kubeeye/clusterinsight_controller.go

```diff
@@ -114,6 +116,19 @@ func (r *ClusterInsightReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	// fill
 	clusterInsight.Status.ScoreInfo = scoreInfo
 
+	// get plugins audit result
+	pluginsList := []string{"kubehunter", "kubebench", "kubescape"}
```

Comment:
This list of plugins should be defined by `clusterInsight.Spec.Plugins`, not hardcod.